sql/table_test.go

```diff
@@ -101,7 +102,7 @@ func TestMakeTableDescColumns(t *testing.T) {
 			t.Fatalf("%d: %v", i, err)
 		}
 		schema, err := makeTableDesc(stmt[0].(*parser.CreateTable))
-		if err != nil {
+		if err != nil && !testutils.IsError(err, "primary index unspecified") {
```

Comment:
You can't look for a specific err using (err == errors.New("primary index unspecified"))